### ISSUE TITLE
Add Author Column to Recent Commits

### ISSUE THREAD

**Issue by alextricity25** [NeogitOrg/neogit#652]
**Is your feature request related to a problem? Please describe.**
I find myself frequently wanting to see the author when viewing `Recent commits`, but most of the times I'm too lazy to press enter on the commit to inspect it's details. 

**Describe the solution you'd like**
I would like the author of the commit to also be included in `Recent commits` as a separate column so I can quickly see who authored the commit.

**Describe alternatives you've considered**
Start exercising, change my diet, work on my relationships, all in a futile effort to overcome my laziness. 

**Additional context**
<img width="780" alt="image" src="https://github.com/NeogitOrg/neogit/assets/4964964/b111be09-da88-48e3-ae90-7380cb8ea3b5">

**Comment by treatybreaker** [comment]
The place to embed that information is over here: https://github.com/NeogitOrg/neogit/blob/3d63d76f04dc7102c83a057ded30a93822b59dec/lua/neogit/lib/git/log.lua#L273

Or do some custom parsing for the commit in `render_section`, over here: https://github.com/NeogitOrg/neogit/blob/3d63d76f04dc7102c83a057ded30a93822b59dec/lua/neogit/status.lua#L193

Not sure what the best approach on that is. I'm guessing we'd want to set recent commit formatting based on a config flag.

**Comment by treatybreaker** [comment]
I have it working on [this branch](https://github.com/treatybreaker/neogit/tree/feat/author-column-recent-commits). No flags to opt in or out, but this is how it looks:

![image](https://github.com/NeogitOrg/neogit/assets/58627896/416102dc-1458-4985-8e35-6f6fcf5b53d5)

**Comment by alextricity25** [comment]
@treatybreaker you're an absolute legend. If you PR this i'll be more than happy to review and approve.
<img width="831" alt="image" src="https://github.com/NeogitOrg/neogit/assets/4964964/d1a738ae-1eb7-4a14-81c1-418a244e86e9">

**Comment by treatybreaker** [comment]
Yep, definitely going to PR. Writing some tests for it currently 😎.
